lib/reply.js

```diff
@@ -535,6 +535,7 @@ function getHeaderFallback (reply, key) {
   return value
 }
 
+// TODO: check if cleanup/update in this branch, or in the other for code changes with Node.js 8 features ... wip
```

Comment:
We should just remove this block, and rename `getHeaderProper` to `getHeader`

Reply:
sure, I'll do asap